content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/_cq_dialog/.content.xml

```diff
@@ -18,9 +18,9 @@
     jcr:primaryType="nt:unstructured"
     jcr:title="Title"
     sling:resourceType="cq/gui/components/authoring/dialog"
-    extraClientlibs="[core.wcm.components.title.v2.editor]"
+    extraClientlibs="[core.wcm.components.title.v3.editor]"
```

Comment:
Instead of defining a specific client library for Title v3, we should use the client library of v2 (`core.wcm.components.title.v2.editor`) as the logic for the new link attributes is already available in v2.

Reply:
Is it necessary to create an edit dialog for v3? If not, we could remove it and automatically use the one from v2.

Reply:
I think it makes sense the new version is as independent as possible from the previous version(s), just in case we might want to remove the previous versions from the content package (and only ship them in a `-deprecated` package). Hence, I would keep the v3 clientlibs and dialog.

Reply:
Made the changes based on the latest clarifications: we will duplicate HTL scripts and dialogs for the new component versions, but we won't duplicate the clientlibs, instead we will use the former version by referencing the category.